index.yaml

```diff
@@ -809,9 +809,9 @@ entries:
       catalog.cattle.io/release-name: portshift-operator
     apiVersion: v1
     appVersion: v0.1.3
-    created: "2020-10-19T19:35:34.247083742Z"
-    description: 'Portshift cloud-native security platform is an agentless security
-      solution for containerized applications '
+    created: "2021-06-23T17:44:55.488041-07:00"
+    description: |
+      Portshift cloud-native security platform is an agentless security solution for containerized applications
```

Comment:
minor change that generates the description in a new line. The other changes are just timestamp changes.